data/conf/hive-site.xml

```diff
@@ -130,6 +130,21 @@
   <description></description>
 </property>
 
+<property>
+  <name>yarn.app.mapreduce.am.command-opts</name>
+  <value> --add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED --add-opens java.base/java.util.concurrent.atomic=ALL-UNNAMED --add-opens=java.base/java.util.regex=ALL-UNNAMED </value>
```

Comment:
wonder if we could extract repeating jvm configs into XML entity and reference here

Reply:
````
<!DOCTYPE configuration [
  <!ENTITY java_add_opens "--add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED --add-opens java.base/java.util.concurrent.atomic=ALL-UNNAMED">
]>


<property>
    <name>yarn.app.mapreduce.am.command-opts</name>
    <value>&java_add_opens;</value>
  </property>
````

Reply:
or follow the suggestion in https://github.com/apache/hive/pull/5404/files#r1728780434